<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btSearch">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listAllDevWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>171</width>
         <height>341</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(243, 211, 255);
color: rgb(61, 32, 255);
font: 14pt &quot;.SF NS Text&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetAllTitle">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(243, 211, 255);
color: rgb(61, 32, 255);
font: 22pt &quot;.SF NS Text&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetWord">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(36, 4, 75);
font: 23pt &quot;.SF NS Text&quot;;</string>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>234</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
